lib/commands/run-api-scenario.ts

```diff
@@ -124,14 +102,31 @@ export const builder: yargs.CommandBuilder = {
 
 export async function handler(argv: yargs.Arguments): Promise<void> {
   await cliSuppressExceptions(async () => {
-    const scenarioFilePath = path.resolve(argv.apiScenario);
-    const swaggerFilePaths = getSwaggerFilePathsFromApiScenarioFilePath(scenarioFilePath);
-    if (swaggerFilePaths.length === 0) {
-      throw new Error(
-        `Failed to run api scenario: Could not find related swagger file. ${scenarioFilePath}`
-      );
+    const scenarioFilePath = pathResolve(argv.apiScenario);
```

Comment:
Previously, we read swagger from from apiScenario path. But current behavior seems read swagger files from readme tag. Is this expected? We choose readme tag as swagger input, right?

Reply:
yes, this is expected, because we need support loading from remote uri, in which case we cannot scan the folder structure but need rely on the readme tag to find swagger file.